api/controllers/site-user.js

```diff
@@ -0,0 +1,29 @@
+const siteSerializer = require('../serializers/site');
```

Comment:
What if instead of naming this API route `site-user`, we did something like `site/:id/notifications`? It could all work the same, but with a different name it may be a little bit easier to parse out what this API route does.

Reply:
done